lms/djangoapps/discussion_api/tests/utils.py

```diff
@@ -74,6 +74,17 @@ def register_get_user_response(self, user, subscribed_thread_ids=None, upvoted_i
             status=200
         )
 
+    def register_subscription_response(self, user):
+        """
+        Register a mock response for POST on the CS user subscription endpoint
+        """
+        httpretty.register_uri(
+            httpretty.POST,
+            "http://localhost:4567/api/v1/users/{id}/subscriptions".format(id=user.id),
```

Comment:
"http://localhost:4567/" is the default prefix for accessing the comments service (see lms/lib/comment_client/settings.py)